staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -1071,6 +1073,8 @@ type NetworkDeviceData struct {
 	// associated subnet mask.
 	// e.g.: "192.0.2.5/24" for IPv4 and "2001:db8::5/64" for IPv6.
 	//
+	// Must not contain more than 16 entries.
+	//
```

Comment:
Can you define Go constants for all things checked by validation?

Without having looked at the validation code, I see at least these:
- max size of the `Conditions` slice
- max size of `Data` buffer
- max size of this `IPs` slice

Reply:
... and of course, use those constants in the validation code.

Reply:
Sorry, I missed that part, fixed now